cartography/intel/gcp/compute.py

```diff
@@ -122,14 +228,15 @@ def load_gcp_instances(neo4j_session, data, gcp_update_tag):
     :return: Nothing
     """
     query = """
-    MATCH (p:GCPProject{id:{ProjectId}})
+    MERGE (p:GCPProject{id:{ProjectId}})
     MERGE (i:Instance:GCPInstance{id:{PartialUri}})
     ON CREATE SET i.firstseen = timestamp()
```

Comment:
partial_uri should be in the ON CREATE SET since it's the id map and will never change.

Reply:
Fixed